host-bin/enter-chroot

```diff
@@ -385,10 +385,120 @@ if ! mountpoint -q "$destmedia"; then
     mount --rbind /media "$destmedia"
 fi
 
-# Bind-mount ~/Downloads if we're logged in as a user
-localdownloads='/home/chronos/user/Downloads'
-if [ -z "$NOLOGIN" -a -n "$CHROOTHOME" -a -d "$localdownloads" ]; then
-    bindmount "$localdownloads" "$CHROOTHOME/Downloads" exec
+# Provide a default /etc/crouton/shares file
+shares="`fixabslinks '/etc/crouton/shares'`"
+if [ -e "$shares" -a ! -f "$shares" ]; then
+    echo "Not mounting shares: /etc/crouton/shares is not a file." 1>&2
+elif [ ! -f "$shares" ]; then
+    cat > "$shares" <<EOF
+# Defines directories to map into the chroot from the host.
+# Order is important: be sure to mount directories before you mount
+# subdirectories of those mounts.
+# The only directories that can be mapped from outside of the chroot are
+# subdirectories of the following locations:
+#  downloads: ~/Downloads
+#  shared: /mnt/stateful_partition/crouton/shared
+#  encrypted: ~/crouton/shared
+# Add a line with a single keyword to modify the mount options of all the bind
+# mounts that follow. For example, the following will make all following mounts
```

Comment:
This feels error-prone, I'd rather specify the options on each line (default to exec is good)

Sorry, again, I'm a bit late in the discussion, but I'd prefer a `fstab`-like syntax, without the arrow, e.g.:
```
downloads ~/Downloads exec
shared/home ~/Share noexec,ro
```